paddle/fluid/operators/activation_op.h

```diff
@@ -49,6 +76,27 @@ class ActivationKernel
   }
 };
 
+template <typename Functor>
```

Comment:
Could `MKLDNNActivationKernel` and `MKLDNNActivationGradKernel` move to a new file named `mkldnn_activation_op.h`?

Reply:
Done.